<launch>
    <rosparam command="load" file="$(find campus_driving)/params/calibParam.yaml" />

	<node pkg="campus_driving" name="double_antenna" type="double_antenna.py"  />
  <node pkg="campus_driving" name="angleSensor" type="angleSensor.py"  />
	<node pkg="campus_driving" name="M30" type="M30.py"  />
	<node pkg="campus_driving" name="CAN_receive_test" type="CAN_receive_test.py"  />
	<node pkg="campus_driving" name="CAN_Driver" type="CAN_Driver.py"  />
  <node pkg="campus_driving" name="cloudClient" type="cloudClient.py"  />
	<node pkg="campus_driving" name="vehicle_state" type="vehicle_model.py"  />
	<node pkg="campus_driving" name="local_planning" type="local_planning.py" />
  <node pkg="campus_driving" name="safeArea" type="safeArea.py"  />
	<node pkg="campus_driving" name="steer_control" type="steer_control.py" output="screen"/>
  <include file="$(find gridmap)/launch/chushandian_multi.launch" />
  <node pkg="campus_driving" name="display" type="display.py" />
  <node pkg="campus_driving" name="rvizSet" type="rvizSet.py" />
  <node pkg="campus_driving" name="logTopic" type="logTopic.py" />

	<node pkg="tf" name="marker2map" type="static_transform_publisher" args = "0 0 0 0 0 0 /map /gps_marker 10" />
	<node pkg="rviz" type="rviz" name="road_display" args="-d $(find campus_driving)/csd.rviz" />


</launch>
